docs/contents/news/1-19/1/CHANGELOG-1.19.1.md

```diff
@@ -0,0 +1,52 @@
+# Changelog for v1.19.1
+
+This changelog highlights the changes for [v1.19.1](https://github.com/aws/eks-distro/tree/v1-19-eks-1).
+
+## Upgrade Kubernetes Versions
+
+* Upgraded Kubernetes to 1.19
+* Upgraded dependencies and components
```

Comment:
Did we actually do this for any deps?

Reply:
...no. Changing now and adding version for components